pkg/controller/manager/clusternetwork/controller.go

```diff
@@ -137,6 +153,26 @@ func (h Handler) initializeClusterNetwork() error {
 	return nil
 }
 
+func (h Handler) initializeUntaggedNetwork() error {
+	untaggedNetworkConfig := fmt.Sprintf(`{"cniVersion":"0.3.1","name":"%s","type":"bridge","bridge":"%s-br","promiscMode":true,"ipam":{}}`,
```

Comment:
I will suggest to gen the bridge name before the config string
If we change the bridge name rule. we could modify the naming rule.
For example, change the posfix of bridge name.
and we don't need to change here to match the new rule